src/kibana/plugins/vis_types/controls/radius_ratio_option.html

```diff
@@ -0,0 +1,16 @@
+<div>
+  <span class="hintbox_label" ng-click="showDotRatioHint = !showDotRatioHint">
```

Comment:
Confused by the use of the `hintbox` classes in this context. There is a css class for `hintbox-label` (not `hintbox_label`) but it doesn't seem to really relate to the dot size ratio at all.